connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -334,6 +336,57 @@ export async function confluenceUpsertPageActivity({
   await upsertConfluencePageInDb(connector.id, page, visitedAtMs);
 }
 
+export async function confluenceUpdatePagesParentIdsActivity(
```

Comment:
Why are we doint it in two passes? can't we get the info from the get go at initial upsert time?

Reply:
We do it at the very end once we imported all the pages to avoid endless loop like it happened on Notion. You can't know all the parents that a Confluence page has until you imported all pages from a space.

Reply:
Roger that